autodarts-caller.py

```diff
@@ -807,6 +808,11 @@ def play_sound_effect(sound_file_key, wait_for_last = False, volume_mult = 1.0,
         play_sound(random.choice(caller[sound_file_key]), wait_for_last, volume_mult, mod)
         return True
     except Exception as e:
+        if WEB == 0 or WEB == 2:
+            #To ensure that the next files are not being played when a walk on player file is missing
+            if wait_for_last == True and WALK_ON:
```

Comment:
Do not modify this universal function. Please find a way outside this function

Reply:
removed, now send true as wait for last instead of the last play_sound_effect